.github/scripts/unittest.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+./.github/scripts/setup-env.sh
+
+# Prepare conda
+CONDA_PATH=$(which conda)
+eval "$(${CONDA_PATH} shell.bash hook)"
+conda activate ci
```

Comment:
@osalpekar https://github.com/pytorch/vision/pull/7189#discussion_r1112842083 becomes even more relevant now. Without it, we need to repeat the top two lines everywhere. I'll get on it.